config-gen/examples/kustomize/README.md

```diff
@@ -0,0 +1,46 @@
+# Kustomize Example
+
+This directory contains an example of using `kubebuilder alpha config-gen` as a Kustomize transformer
+plugin.
+
+This enables using `config-gen` in traditional kustomize workflows (patch, bases, etc).
+
+## Summary
+
+`config-gen` may be used from `kustomize` as a transformer plugin.  This allows the output
+to be customized using `commonLabels`, `commonAnnotations`, `namespace`, etc.
```

Comment:
I think this will be an incredibly common use case. I wonder if the setup can be streamlined (not necessary right now since config-gen is alpha).

Reply:
Done

Reply:
Added a subcommand to install the plugin